scripts/extractor.js

```diff
@@ -91,9 +92,25 @@ module.exports.run = function run() {
 
       saveLeaderboardRatingsToFile(ratings, "members/2v2.json");
     });
+    
+    models.reset();
+
+    request(process.env.API_URL + "/data/leaderboardRating?include=player&sort=-rating&filter=leaderboard.id==4;updateTime=ge=" +
+      pastMonth.format("YYYY-MM-DDTHH:mm:ss") + "Z", function (error, response, body) {
+      if (error || response.statusCode > 210) {
```

Comment:
is that the traditional NPM request we're using here? It's heavily deprecated if so, we should probably not use that anymore (2020)
probably not in the scope of that PR but good to keep in mind